action.yml

```diff
@@ -43,6 +43,12 @@ runs:
         $statusCode = 0
         $message = ""
         try {
+          Write-Host "Parsing ${{ inputs.comment_body }}"
+          if ("${{ inputs.comment_body }}" -match "@(?<botName>.+) backport (?<branchName>.+)") {
+            $botName = $Matches.botName;
+            $backportTargetBranch = $Matches.branchName;
+          }
```

Comment:
Nit: I really hope we don't have spaces in our branch names, but we might want to make sure we account for those properly. Wanted to make sure the following input is an error mode:
`@vsmacbot backport main please`

```pwsh
if ("${{ inputs.comment_body }}" -match "@(?<botName>.+) backport (?<branchName>(`".+`")|([^\s]+))$") {
  $botName = $Matches.botName;
  $backportTargetBranch = $Matches.branchName.TrimStart("`"").TrimEnd("`"")
} else {
  throw "Could not parse comment body: ${{ inputs.comment_body }}"
}
```

Reply:
Shouldn't the error be thrown only if we find the botname part?